tools/update-tester/src/testrunner.rs

```diff
@@ -69,6 +98,42 @@ fn connect_to(config: &ConnectionConfig<'_>) -> TestClient {
     TestClient(client)
 }
 
+pub fn update_to_and_validate_new_toolkit_version(root_config: &ConnectionConfig) {
+    // update extension to new version
+    let test_db_name = "tsdb_toolkit_test";
+    let test_config = root_config.with_db(test_db_name);
+
+    let mut test_client = connect_to(&test_config);
+    test_client.update_to_current_version();
+
+    // run validation tests
+    let expected_results: Vec<Vec<Option<String>>> = vec![vec![
```

Comment:
`QueryValues` (line 179)?  Though I'm not a fan of type alias like that.  I think they obscure rather than clarify.